src/components/PressureBar.vue

```diff
@@ -23,12 +23,24 @@ export default {
     },
     geraEstilo (tendencia) {
       return {
-        'progress-down after': true,
-        'progress-up after': true
+        'progress-down after': this.coeficiente < 0,
+        'progress-up after': this.coeficiente >= 0
       }
     }
   },
   computed: {
+    energia () {
+      if (this.listaEnergias[this.id]) { return this.listaEnergias[this.id][0].energia_recente }
+      return 0
+    },
+    coeficiente () {
+      if (this.listaCoeficientes[this.id]) { return this.listaCoeficientes[this.id] }
```

Comment:
mesma coisa do comentário anterior
```suggestion
    return this.listaCoeficientes[this.id] || 0
```